app/settings.py

```diff
@@ -126,7 +126,7 @@ def utcoffset_or_fail(date_value, key):
 EQ_SESSION_ID = "eq-session-id"
 
 EQ_LIST_ITEM_ID_LENGTH = 6
-MAX_NUMBER = 9999999999
+MAX_NUMBER = 900_999_999_999_999
```

Comment:
Should this not be **999**_999_999_999_999?

Reply:
Oops, updated 👍